protos/dcs/tts/v0/tts.proto

```diff
@@ -0,0 +1,82 @@
+syntax = "proto3";
+package dcs.tts.v0;
+import "dcs/common/v0/common.proto";
+option csharp_namespace = "RurouniJones.Dcs.Grpc.V0.Tts";
+option go_package = "github.com/DCS-gRPC/go-bindings/dcs/v0/tts";
+
+service TtsService {
```

Comment:
I wonder if it would be possible to have a `TransmisionInProgress` API call that returns true if _we_ are already transmitting something on a freq. My gut says that this will probably be difficult.